### DISCUSSION TITLE
Dukun langit does not work

### DISCUSSION THREAD

**Discussion by AlexSkyX3** [thatskymod/Sky-CotL-Scripts#89] [General]
I went to play on sky today and just as i activated the mod it did not work, please fix this

**Comment by thatskymod** [comment]
### Update finished
All files from the [Libs/](https://github.com/thatskymod/Sky-CotL-Scripts/tree/main/Canvas/Libs) folder have now been replaced by their newer counterparts, including Dukun Langit.

Thank you for standing by while the issues were being addressed.
